tests/unit/test_ipfs.py

```diff
@@ -87,6 +87,14 @@ def test_get_content(self):
             ).get_content(ACCESS_KEY),
         )
 
+    def test_get_new_content(self):
+        x = "0x22e58ea20cbe7269450637ffc0f40fee7bca0d66fc6bf63666bae4ba08c99688"
+        y = "0x93fc55a145fafb4e65a55f92ea416b55be4b4720c2a2c5c0071b67800c6235b2"
+        z = "0xe0e1f18b98680421df973d2d92d0f1603d35dcad58890f2f9de6ac914bcd54ae"
+        for element in [x, y, z]:
+            content = Cid.fetch_from_backend(element, 1, 10)
+            print(content)
+
```

Comment:
Can you please elaborate how this is relevant to the PR?

Reply:
It isn't, I was testing something else in the same place (when I realized that I needed this new script). Will clean up before merge. The only thing we need here is the delete_from script.